onnxruntime/python/tools/bert/notebooks/PyTorch_Bert-Squad_OnnxRuntime_CPU.ipynb

```diff
@@ -0,0 +1,1150 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "OnnxRuntime cpu Inference time = 91.51 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "import onnxruntime\n",
+    "import numpy\n",
+    "\n",
+    "# Print warning if user uses onnxruntime-gpu instead of onnxruntime package.\n",
+    "if 'CUDAExecutionProvider' in onnxruntime.get_available_providers():\n",
+    "    print(\"warning: onnxruntime-gpu is not built with OpenMP. You might try onnxruntime package to test CPU inference.\")\n",
+    "\n",
+    "sess_options = onnxruntime.SessionOptions()\n",
+    "\n",
+    "# Optional: store the optimized graph and view it using Netron to verify that model is fully optimized.\n",
+    "# Note that this will increase session creation time so enable it for debugging only.\n",
+    "sess_options.optimized_model_filepath = os.path.join(output_dir, \"optimized_model_cpu.onnx\")\n",
+    "   \n",
+    "if use_openmp:\n",
+    "    sess_options.intra_op_num_threads=1\n",
+    "else:\n",
+    "    sess_options.intra_op_num_threads=psutil.cpu_count(logical=True)\n",
+    "\n",
+    "# Specify providers so that even onnxruntime-gpu can be used for CPU inference.\n",
+    "session = onnxruntime.InferenceSession(export_model_path, sess_options, providers=['CPUExecutionProvider'])\n",
+    "\n",
+    "latency = []\n",
+    "for i in range(total_samples):\n",
+    "    data = dataset[i]\n",
+    "    # Use contiguous array as input might improve performance.\n",
+    "    # You can check the results from performance test tool to see whether you need it.\n",
+    "    ort_inputs = {\n",
+    "        'input_ids':  numpy.ascontiguousarray(data[0].cpu().reshape(1, max_seq_length).numpy()),\n",
+    "        'input_mask': numpy.ascontiguousarray(data[1].cpu().reshape(1, max_seq_length).numpy()),\n",
+    "        'segment_ids': numpy.ascontiguousarray(data[2].cpu().reshape(1, max_seq_length).numpy())\n",
+    "    }\n",
+    "    start = time.time()\n",
+    "    ort_outputs = session.run(None, ort_inputs)\n",
+    "    latency.append(time.time() - start)\n",
+    "print(\"OnnxRuntime cpu Inference time = {} ms\".format(format(sum(latency) * 1000 / len(latency), '.2f')))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "***** Verifying correctness *****\n",
+      "PyTorch and ONNX Runtime output 0 are close: True\n",
+      "PyTorch and ONNX Runtime output 1 are close: True\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"***** Verifying correctness *****\")\n",
+    "for i in range(2):\n",
+    "    print('PyTorch and ONNX Runtime output {} are close:'.format(i), numpy.allclose(ort_outputs[i], outputs[i].cpu(), rtol=1e-05, atol=1e-04))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. Offline Optimization Script and Test Tools\n",
```

Comment:
Change the header size to "##" to align with the other numbered steps. Also you may add "[OPTIONAL]" to it because it seems to be intended for users who want to know ort bert optimization in extensive detail?